src/sentry/api/endpoints/chunk_upload.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import absolute_import
+
+# from rest_framework.permissions import IsAuthenticated
+from rest_framework import status
+from rest_framework.response import Response
+
+from sentry.models import FileBlob
+from sentry.models.file import DEFAULT_BLOB_SIZE
+from sentry.api.base import Endpoint
+from sentry.api.bases.project import ProjectReleasePermission
+
+
+UPLOAD_ENDPOINT = 'https://sentry.io'
+MAX_CHUNKS_PER_REQUEST = 16
+MAX_CONCURRENCY = 4
+HASH_ALGORITHM = 'sha1'
+
+
+class ChunkUploadEndpoint(Endpoint):
+    permission_classes = (ProjectReleasePermission, )
+
+    def get(self, request):
+        return Response(
+            {
+                'url': UPLOAD_ENDPOINT,
```

Comment:
What's the reason for returning this value in the first place? I understand the value in this endpoint being able to fetch this information about basically the server's support, but this in theory just points back to itself? So it's just the exact same endpoint that you hit for GET. In what situation would it be different?

Reply:
As I said before the idea is to maybe handle the upload through a different endpoint e.g.: `upload.sentry.io` which has some kind of keep alive or whatever.
I am not really sure if this endpoint basically is just another instance of Sentry with different webserver settings.